src/lib/tooltip/tooltip.ts

```diff
@@ -58,7 +58,11 @@ export function throwMdTooltipInvalidPositionError(position: string) {
   selector: '[md-tooltip], [mdTooltip], [mat-tooltip], [matTooltip]',
   host: {
     '(longpress)': 'show()',
+    '(focus)': 'show()',
+    '(blur)': 'hide(0)',
+    '(keydown.esc)': 'hide(0)',
```

Comment:
Can't use `keydown.xxx` syntax because it's not supported in Safari 9 (Angular uses `KeyboardEvent.key` for this). You'll need to just handle `keydown` and check the keycode. 

You'll also need to check whether the tooltip is actually present, and call `stopPropagation` if it. This is necessary in cases where you have something like a tooltip inside of a dialog; you only want to dismiss the tooltip without closing the dialog.